pbsmetrics/go_metrics_test.go

```diff
@@ -179,6 +184,160 @@ func TestRecordBidTypeDisabledConfig(t *testing.T) {
 	}
 }
 
+func TestRecordDNSTime(t *testing.T) {
+	type testIn struct {
+		dnsLookupDuration time.Duration
+	}
+	type testOut struct {
+		expDuration time.Duration
+	}
+	testCases := []struct {
+		description string
+		in          testIn
+		out         testOut
+	}{
+		{
+			description: "Five second DNS lookup time",
+			in: testIn{
```

Comment:
What's the reason behind wrapping the `dnsLookupDuration` and `expDuration` in `testIn` and `testOut` structs?

Reply:
Makes sense. Modified